eZ/Publish/API/Repository/Exceptions/ContentFieldValidationException.php

```diff
@@ -13,7 +13,7 @@
 /**
  * This Exception is thrown on create or update content one or more given fields are not valid.
  */
-abstract class ContentFieldValidationException extends ForbiddenException
+abstract class ContentFieldValidationException extends ContentValidationException
```

Comment:
Pushed the update as discussed: https://github.com/ezsystems/ezpublish-kernel/pull/1734/commits/51a84df9d587364a00e7ea4b1d640445cda63150